database/mssql/scripts/versions/v_2_ddl.sql

```diff
@@ -155,6 +155,7 @@ INSERT [dbo].[ORBC_POWER_UNIT_TYPE] ([POWER_UNIT_TYPE], [TYPE], [DESCRIPTION], [
 INSERT [dbo].[ORBC_POWER_UNIT_TYPE] ([POWER_UNIT_TYPE], [TYPE], [DESCRIPTION], [SORT_ORDER], [IS_ACTIVE], [CONCURRENCY_CONTROL_NUMBER], [DB_CREATE_USERID], [DB_CREATE_TIMESTAMP], [DB_LAST_UPDATE_USERID], [DB_LAST_UPDATE_TIMESTAMP]) VALUES (N'LOGGING', N'Logging Trucks', N'Logging Truck is a truck or truck and trailer combination used to haul, in their natural state, green felled or bucked logs or poles.', NULL, 1, NULL, N'dbo', GETUTCDATE(), N'dbo', GETUTCDATE())
 INSERT [dbo].[ORBC_POWER_UNIT_TYPE] ([POWER_UNIT_TYPE], [TYPE], [DESCRIPTION], [SORT_ORDER], [IS_ACTIVE], [CONCURRENCY_CONTROL_NUMBER], [DB_CREATE_USERID], [DB_CREATE_TIMESTAMP], [DB_LAST_UPDATE_USERID], [DB_LAST_UPDATE_TIMESTAMP]) VALUES (N'LOGOFFH', N'Logging Trucks - Off-Highway', N'Logging Trucks - Off-Highway are permitted on an occasional basis from manufacturers to/from customers and to/from the bush for repairs etc., at maximum dimensions noted in Policy.', NULL, 1, NULL, N'dbo', GETUTCDATE(), N'dbo', GETUTCDATE())
 INSERT [dbo].[ORBC_POWER_UNIT_TYPE] ([POWER_UNIT_TYPE], [TYPE], [DESCRIPTION], [SORT_ORDER], [IS_ACTIVE], [CONCURRENCY_CONTROL_NUMBER], [DB_CREATE_USERID], [DB_CREATE_TIMESTAMP], [DB_LAST_UPDATE_USERID], [DB_LAST_UPDATE_TIMESTAMP]) VALUES (N'LWBTRCT', N'Long Wheelbase Truck Tractors Exceeding 6.2 m up to 7.25 m', N'Long Wheelbase Truck Tractors Exceeding 6.2 m up to 7.25 are vehicles that do not comply with wheelbase requirements indicated in CTR Appendix B', NULL, 1, NULL, N'dbo', GETUTCDATE(), N'dbo', GETUTCDATE())
+INSERT [dbo].[ORBC_POWER_UNIT_TYPE] ([POWER_UNIT_TYPE], [TYPE], [DESCRIPTION], [SORT_ORDER], [IS_ACTIVE], [CONCURRENCY_CONTROL_NUMBER], [DB_CREATE_USERID], [DB_CREATE_TIMESTAMP], [DB_LAST_UPDATE_USERID], [DB_LAST_UPDATE_TIMESTAMP]) VALUES (N'MUNFITR', N'Municipal Fire Trucks', N'', NULL, 1, NULL, N'dbo', GETUTCDATE(), N'dbo', GETUTCDATE())
```

Comment:
Please create v20 for the change

Reply:
Kindly coordinate with John Fletcher for DB change, versioning and revert script

Reply:
Ok I'll ask John Fletcher

Reply:
Moved to v20.